app/assets/javascripts/discourse/app/components/watch-read.js

```diff
@@ -1,3 +1,4 @@
+import { bind } from "discourse-common/utils/decorators";
```

Comment:
watch read is not related to element in viewport though? Or Am I missing something?

Reply:
Yeah, have now split it out into a separate PR: https://github.com/discourse/discourse/pull/17001

Reply:
FTR I didn't really care if it was in the same, it was more to be sure you didn't make a mistake and also maybe change title :p